filters-share/specific_CSS.txt

```diff
@@ -5,6 +5,7 @@ loawa.com#$#div[id*="ad_"][style*="position: absolute; top:"] { top: 100% !impor
 loawa.com#$#div[class*="p-0 m-0 mb-2 d-none d-md-block text-center "] { position: relative !important; top: -200px !important; }
 loawa.com#$##ad_center_v.adsense-ch { position: relative !important; top: 420px !important; }
 !
+sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
```

Comment:
Does this rule make dcinside login page looks like better?

Reply:
![image](https://user-images.githubusercontent.com/34261355/170831274-2d69b203-cbbe-4817-ae24-93abe9c2eb2c.png)

@piquark6046

Reply:
Did you test the rule on a chromium browser?
@green1052

Reply:
<details>
<summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/34261355/170831302-b46d30f3-71da-491f-957b-f671e201cf8f.png)

</details>

with two rules
```
sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
sign.dcinside.com##.ban_box
```

Reply:
> Did you test the rule on a chromium browser?

yeah i tested in ungoogled chromium

Reply:
@green1052 
With AdGuard for Windows?

Reply:
nop ublock

Reply:
Please test new rule on a chromium browser.

Reply:
sadly i turn off my pc

Reply:
![Screenshot_20220529-004406_Kiwi Browser](https://user-images.githubusercontent.com/34261355/170832573-b61db984-a5a8-483f-952d-28693ba0ff3d.jpg)

![Screenshot_20220529-004417_AdGuard](https://user-images.githubusercontent.com/34261355/170832574-15abe324-17d0-4f03-a852-59c7f44d99b3.jpg)

Reply:
@piquark6046

Reply:
No. You have to add the following rules when you test on a chromium browser:
```
sign.dcinside.com#$#.login_inputbox { width: 100% !important; }
sign.dcinside.com#$#.login_inputbox .inner { width: calc(100% - 115px) !important; }
sign.dcinside.com##.ban_box
``

Reply:
![Screenshot_20220529-004924_Kiwi Browser](https://user-images.githubusercontent.com/34261355/170832786-75888154-cf2e-4481-8e46-51d50d89bc70.jpg)

its working sorry